model/connect/PDOConnector.php

```diff
@@ -227,7 +228,7 @@ public function exec($sql, $errorLevel = E_USER_ERROR) {
 		$result = $this->pdoConnection->exec($sql);
 
 		// Check for errors
-		if ($result !== false) {
+		if (!$this->hasError($result)) {
```

Comment:
The problem is that `false` as a return value now reports hasError() = false, when before it was false. Docs on this method:

```
PDO::exec() returns the number of rows that were modified or deleted by the SQL statement you issued. If no rows were affected, PDO::exec() returns 0.
```

I think the `!== false` check was correct.